eduroam.md

```diff
@@ -40,98 +53,190 @@ The process of connecting to the network may be different depending on the Linux
 
 - *Identity:* Your RIT email address. i.e. [username]@rit.edu
 - *Authentication:* TLS
-- *Wifi Security:* WPA and WPA2 Enterprise
-- *CA certificate:* The RIT .cer certificate you downloaded earlier.
+- *Wifi Security:* WPA2 Enterprise
+- *CA certificate:* The RIT `.cer` certificate you downloaded earlier.
 - *User Certificate/private key:* The other file you downloaded (**not the .cer file**). Note that, depending on the file you download, the certificate and private key may or may not be separate.
 - *User Key Password:* The password you set for the above file earlier.
 
+## GUI Setup
+
+This section will work with both `.pem` and `.p12` certificates and keys.
 
-### NetworkManager (GUI) Setup
+### NetworkManager Configuration
 
-1. To connect click the wifi logo in the status bar, and then select 'eduroam' ![](/assets/img/eduroam/open-networkmanager.png)
+1. To connect click the wifi logo in the status bar, and then select `eduroam`. ![](/assets/img/eduroam/open-networkmanager.png)
 
 2. Enter the configuration details in the window that appears. If the password field turns red it means the password is incorrect. ![](/assets/img/eduroam/configure-networkmanager.png)
 
-Once these instructions are complete, you will be able to connect to the
-new eduroam setup without needing to run any scripts!
+Your computer should now be connected to the `eduroam` network.
+
+## CLI Setup
+
```

Comment:
do you think it makes sense for this section to have its own separate page? (or for this page to otherwise be broken up? maybe by each different method (networkmanager, wpa_supplicant .etc)?

Reply:
I would absolutely love to separate this page out, and by individual network manager solution would probably be best (networkmanager-gui, nmcli, iwd, wpa). I wanted to first get the updated information out, as I knew I could easily fix that first. Once the security requirements for ITS are patched, I will look more into separating them out into different pages.